consensus/src/chained_bft/event_processor.rs

```diff
@@ -516,38 +497,11 @@ impl<T: Payload> EventProcessor<T> {
             Ok(vote) => vote,
         };
 
-        // Safety invariant: The vote being sent is for the proposal that was received.
-        debug_checked_verify_eq!(proposal_id, vote.vote_data().proposed().id());
-        // Safety invariant: The last voted round is updated to be the same as the proposed block's
-        // round. At this point, the replica has decided to vote for the proposed block.
-        debug_checked_verify_eq!(
-            self.safety_rules
-                .consensus_state()
-                .unwrap()
-                .last_voted_round(),
-            proposal_round
-        );
-        // Safety invariant: qc_parent <-- qc
-        // the preferred block round must be at least as large as qc_parent's round.
-        debug_checked_verify!(
```

Comment:
the intern would be sad to see this

Reply:
Hahaha, spoke offline with @dmitri-perelman, we agreed it wasn't adding much value. Then if you take a look, I think these checks became irrelevant as we eliminated extra pieces of data within the block late last year.